### ISSUE TITLE
Card Radius on Widget too big

### ISSUE THREAD

**Issue by newhinton** [stefan-niedermann/nextcloud-notes#1441]
Please use **[GitHub reactions](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)** 👍 to show that you are affected by the same issue. Please don't comment if you have no relevant information to add!

**Describe the bug**

The Radius is too big to look good. It comes too close to the content in the corners which looks strange

**To Reproduce**
1. Create Widget with text

**Expected behavior**

A smaller radius would be better

**Screenshots**

![Screenshot_20220104-113544_Trebuchet](https://user-images.githubusercontent.com/25279821/148046790-ebb5bd93-834f-4317-a065-d00f76a3a57c.png)

**Smartphone (please complete the following information):**
 - Nextcloud Notes-Version (android app): 3.4.18
 - F-Droid or Play Store: FDroid
 - Android-Version: Lineageos 18.1
 - Device: 

**Server**
 - Nextcloud version: 
 - Nextcloud Notes version (server app): 

**Stacktrace**

**Comment by stefan-niedermann** [comment]
I agree that it looks like shit, but that's what Android does starting with version 12 (they called it ["improvements"](https://developer.android.com/about/versions/12/features/widgets)).

Since they decided to just crop the widgets to this huge border radius and i won't maintain different widgets per Android version, we'll have to stick with it.

**Comment by newhinton** [comment]
I feared something like that was the reason for the change. Maybe increasing margin_left/margin_right to fit the bottom and top margins will make it look better?

The examples dont look too bad, but they use icons in the corner which helps to hide the issue a bit

**Comment by newhinton** [comment]
On a sidenote, will you support the new compound buttons so that users can change checkbox states from the widget?

**Comment by jancborchardt** [comment]
> Maybe increasing margin_left/margin_right to fit the bottom and top margins will make it look better?

Yeah, that seems like a good improvement.

Also, is it possible to make the top and bottom behave like padding and not like margin – so that the text is only cut off at the widget top and bottom, and not before?

**Comment by abdusalamApps** [comment]
It looks fine on OnePlus Nord with 6.44 inch screen, but looks like the picture provided in the issue on Galaxy S9 with 5.8 inch screen.
However, I created a pull request for this issue. I hope it gets approved and merged
Here: https://github.com/stefan-niedermann/nextcloud-notes/pull/1588
